packages/app/pages/Members/index.tsx

```diff
@@ -8,7 +8,7 @@
 
 import * as React from 'react'
 import ProjectTemplate from '@vflows/views/templates/ProjectTemplate'
-import { AgentType } from '@vflows/bindings/agent/agent'
+import { AgentType } from '@vflows/bindings/Agent/agent'
```

Comment:
(Not a comment specifically on this PR)
We should find a standard for naming folders. I encountered difficulties with Agent as well in my PRs where there becomes duplicate code when merging.

Reply:
this was changed to match the standards of Ui bindings